test/e2e/scheduling/preemption.go

```diff
@@ -95,10 +95,13 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 		framework.ExpectNoError(err)
 	})
 
-	// This test verifies that when a higher priority pod is created and no node with
-	// enough resources is found, scheduler preempts a lower priority pod to schedule
-	// the high priority pod.
-	ginkgo.It("validates basic preemption works", func() {
+	/*
+		Release : v1.16
+		Testname: Scheduler, Basic Preemption
+		Description: When a higher priority pod is created and no node with enough resources is found,
+		scheduler MUST preempt a lower priority pod and schedule the high priority pod.
+	*/
+	framework.ConformanceIt("validates basic preemption works", func() {
```

Comment:
Looking at the code for this test, the calculation to request 40% of allocatable memory is not really a safe check.  It should be subtracting the currently scheduled CPU requests on that node from allocatable, then taking 40%.

Reply:
> It should be subtracting the currently scheduled CPU requests on that node from allocatable, then taking 40%.

@smarterclayton in origin, we [do currently do that](https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/test/e2e/scheduling/preemption.go#L111-L117), but it still flakes (if you look at my [origin PR above](https://github.com/openshift/origin/pull/23758), I temporarily attempt to revert those changes). I also have had a PR open for a while now in kubernetes to update this as well: https://github.com/kubernetes/kubernetes/pull/82350